src/tests/containers/EventInfo.test.js

```diff
@@ -11,7 +11,7 @@ describe('Event Info', () => {
       slug: 'weekendcollaboration'
     },
     fetchEventInfo: jest.fn(),
-    setLastLocation: () => {},
+    setLastLocation: () => { },
```

Comment:
What is motivating all of these?

Reply:
`setLastLocation` should return a string, not an object
it was added to redirect the user back to the page they were on after successfully logging in

Reply:
@mattwr18 Didn't know that. That seems like an issue with pre-existing code.

What I meant is, what is motivating making all the empty object literals fat -- `{ }`. It doesn't appear to be a linting rule, and there are no fat object literals in the codebase prior to this PR.

Reply:
I guess this is some local linter rule that should never get to this diff.